demo/index.html

```diff
@@ -45,11 +45,33 @@
         margin-bottom: 0;
         padding: 10px 5px;
     }
+    #cmdoutput {
+		padding: 5px;
+		text-align:left;
+		border: 2px solid grey;
+		margin: 5px;
+		display:inline-block;
+	}
+
+	/* Attempt to colour <option> in dropdown selection.
+	colors copied from SCSS (not sure how to access them here)
+	Only one of the following two blocks is needed. 
+	Works with Chrome, but not Firefox; */
+	.c_success{background-color: #51A351 !important;}
+	.c_info{background-color: #2F96B4 !important;}
+	.c_warning{background-color: #f87400 !important;}
+	.c_error{background-color: #BD362F !important;}
+
+	select option[value="success"]  { background: #51A351; }
+	select option[value="info"] 	{ background: #2F96B4; }
+	select option[value="warning"]  { background: #F87400; }
+	select option[value="error"] 	{ background: #BD362F; }
```

Comment:
Instead of SCSS variables you could use CSS variables like this : 
```css
:root {
    --color-name: #51A351;
    --another-color: red;
}
```
That way you should be able to access it in the html like this : 
```css
background-color: var(--color-name);
```

And I think you should use the second option with `select option`, in this case you won't need create CSS class 

For me this is working in Firefox 😉

Reply:
I've made the changes that you suggested. Thanks for the instruction about CSS variables.
I think everything is updated on my end.

For the Readme.md can you also fix the "build/unknown"  button?